Core/Python/get_group_list.py

```diff
@@ -32,38 +32,55 @@
    python get_group_list.py --ip <xx> --user <username> --password <pwd>
 """
 import json
-import sys
 import argparse
 from argparse import RawTextHelpFormatter
 import urllib3
 import requests
+import pprint
 
 
-def get_group_list(ip_address, user_name, password):
+def get_group_list(ome_ip_address, user_name, password):
     """ Authenticate with OME and enumerate groups """
     try:
-        session_url = 'https://%s/api/SessionService/Sessions' % (ip_address)
-        group_url = 'https://%s/api/GroupService/Groups' % (ip_address)
+        session_url = 'https://%s/api/SessionService/Sessions' % ome_ip_address
         headers = {'content-type': 'application/json'}
         user_details = {'UserName': user_name,
                         'Password': password,
                         'SessionType': 'API'}
+        group_data = None
+        next_link_url = 'https://%s/api/GroupService/Groups' % ome_ip_address
 
         session_info = requests.post(session_url, verify=False,
                                      data=json.dumps(user_details),
                                      headers=headers)
         if session_info.status_code == 201:
             headers['X-Auth-Token'] = session_info.headers['X-Auth-Token']
-            print (headers['X-Auth-Token'])
-            response = requests.get(group_url, headers=headers, verify=False)
-            if response.status_code == 200:
-                print (json.dumps(response.json(), indent=4, sort_keys=True))
-            else:
-                print ("Unable to retrieve group list from %s" % (ip_address))
-        else:
-            print ("Unable to create a session with appliance %s" % (ip_address))
-    except:
-        print ("Unexpected error:", sys.exc_info()[0])
+            print(headers['X-Auth-Token'])
```

Comment:
Can we remove this statement - may be good enough to print out that authentication succeeded instead of the token value

Reply:
Sure - I just had it there because it was there originally hahaha

Reply:
Updated